### PR TITLE
adding scenarios created by oreilly

### PR DESCRIPTION
**Pull request by SpicyThrow**
Multi-regio scenarios created by the O'Reilly team.

### CODE REVIEW

File: oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md:3]: "we have grasped" -> "you have seen"

Comment (thatnerd) [reply oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md:3]: In general, try to keep it to second-person rather than first-person plural.
we -> you
our -> your
"let us" -> ""
"let's" -> ""
etc.

File: oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md:7]: Second person here

File: oreilly-scenarios/cockroach-with-friendpay/configure-environment.sh
Comment (thatnerd) [inline oreilly-scenarios/cockroach-with-friendpay/configure-environment.sh:6]: Do you need the geoindex functionality? Does the example run without it? Probably not worth taking out.

File: oreilly-scenarios/cockroach-with-friendpay/exploring-friendpay.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-with-friendpay/exploring-friendpay.md:36]: Have you explained what `jq` does? I know it's for manipulating JSON on the command line, just checking that the students have been told.

Comment (thatnerd) [reply oreilly-scenarios/cockroach-with-friendpay/exploring-friendpay.md:36]: Looks like it hasn't been explained yet. I'd recommend telling them something like:

```
To simulate the front-end, we're going to use `jq`, [a command-line tool for working with JSON](https://stedolan.github.io/jq/manual/).
```

File: oreilly-scenarios/cockroach-with-friendpay/friendpay.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-with-friendpay/friendpay.md:15]: Give this image some alt text that describes it. Maybe just `Cluster HOST`

File: oreilly-scenarios/cockroach-with-friendpay/intro.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-with-friendpay/intro.md:6]: Change bullet 2 to something like "Run the FriendPay application, a lightweight toy payment app."

File: oreilly-scenarios/cockroach-with-friendpay/intro.md
Comment (thatnerd) [inline oreilly-scenarios/cockroach-with-friendpay/intro.md:7]: Change to "See how an application might use CockroachDB in production.

**Review by thatnerd — CHANGES_REQUESTED** [review]
Added some comments that should be considered before approval, though I won't push hard if some (or most) get rejected. Only essential thing is the `.DS_Store` files that need deleting.

**Comment by thatnerd** [comment]
LGTM after my conversation with Crossman & Meagan.

**Comment by SpicyThrow** [comment]
Discussed possible edits with Meagan Goldman and Will but was advised to merge since this had prior approval.

**Pull request merged**
